### ISSUE TITLE
Can not resize columns in table editor on Safari

### ISSUE THREAD

**Issue by Cydiater** [supabase/supabase#11937]
# Bug report

## Describe the bug

Resizing table columns not work on Safari.

## To Reproduce

Just directly go to the table editor page of the studio and check it on Safari.

## Expected behavior

When cursor hang over the dividing line between columns, the cursor should turn into a resizing cursor and able to resize the columns. 

## Screenshots

Here is a short video demonstrating that columns can not be resized on Safari, but can be resized on Chrome.

https://user-images.githubusercontent.com/16633622/214783775-6f1bb729-b41c-4ba0-a935-52ead13083a0.mov

## System information

- OS: macOS 13.1
- Browser: Safari
- Studio docker image: supabase/studio:20221214-4eecc99

**Comment by raph941** [comment]
Hello @Cydiater, 
I attempted to replicate the issue on my end and noticed that the resize function does work on Safari (as shown below). However, unlike Chrome, no resize icon is visible when hovering over the edge of the table, making it less noticeable. 

Please verify if this is also the case for you.

https://user-images.githubusercontent.com/45232708/214841953-2120df4a-7af1-4afa-9d29-02930fa846e7.mov

**Comment by Cydiater** [comment]
Yes, you are right! It is just not showing resize icon on Safari. Thanks for pointing that out. But I believe it is still a bug for not showing the icon on Safari.

**Comment by raph941** [comment]
I agree it's a bug that needs to be looked into. I just wanted to understand the behavior better 👍.

**Comment by alexandregiordanelli** [comment]
https://github.com/supabase/supabase/issues/11981

**Comment by Anikets08** [comment]
Let me know if this is still on
